app/src/main/java/com/github/se/polyfit/data/remote/firebase/PostFirebaseRepository.kt

```diff
@@ -19,25 +21,35 @@ import java.io.ByteArrayInputStream
 import java.io.ByteArrayOutputStream
 import java.util.UUID
 import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.Deferred
 import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.flow.Flow
-import kotlinx.coroutines.flow.flow
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.tasks.await
 import kotlinx.coroutines.withContext
 
 class PostFirebaseRepository(
-    private val db: FirebaseFirestore = FirebaseFirestore.getInstance(),
```

Comment:
i would suggest to keep the parameters and the variables kinda separated here instead:
```
db: FirebaseFirestore = FirebaseFirestore.getInstance(),
rtdb: FirebaseDatabase =
        FirebaseDatabase.getInstance(
            "https://polyfit-316e8-default-rtdb.europe-west1.firebasedatabase.app/"),
private val pictureDb: FirebaseStorage = FirebaseStorage.getInstance(),
```
I think it makes it easier to read the code

Reply:
I think the database is passed as argument for the test

Reply:
Yeah that makes things a bit cleaner. Thanks for the suggestion.